crates/config/src/fs_permissions.rs

```diff
@@ -53,7 +54,8 @@ impl FsPermissions {
     pub fn find_permission(&self, path: &Path) -> Option<FsAccessPermission> {
         let mut permission: Option<&PathPermission> = None;
         for perm in &self.permissions {
-            if path.starts_with(&perm.path) {
+            let permission_path = dunce::canonicalize(&perm.path).unwrap_or(perm.path.clone());
+            if path.starts_with(permission_path) {
```

Comment:
I'm actually not sure why this is the right fix, because the path should already be canonicalized here

https://github.com/foundry-rs/foundry/blob/5ef0d6f22305b78ed240a7371be57108941f12cc/crates/cheatcodes/src/config.rs#L72-L80

Reply:
> I'm actually not sure why this is the right fix, because the path should already be canonicalized here
> 
> https://github.com/foundry-rs/foundry/blob/5ef0d6f22305b78ed240a7371be57108941f12cc/crates/cheatcodes/src/config.rs#L72-L80

that's the normalized path to the file desired to be loaded in test (by using `vm.readFile`), mind that is passed to `is_normalized_path_allowed` function which checks it against configured `fs_permissions` path (that are not normalized)
https://github.com/foundry-rs/foundry/blob/5ef0d6f22305b78ed240a7371be57108941f12cc/crates/cheatcodes/src/config.rs#L72-L84